src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -84,24 +86,54 @@ def check_metadata(self):
         validate(metadata_for_validation, schema)
         self.check_extracted_metadata(metadata)
 
-    def check_run_conversion(self, nwbfile_path: str):
```

Comment:
2 things here - I thought the name `check_run_conversion` was too generic since it made other assertions inside it besides just ensuring that the call to `.run_conversion` succeeded without error